sdk/appcenter-storage/src/test/java/com/microsoft/appcenter/storage/TestDocument.java

```diff
@@ -0,0 +1,10 @@
+package com.microsoft.appcenter.storage;
+
+public class TestDocument {
```

Comment:
you use that in a lot of places, can you reuse this class in tests?

Reply:
Spoke offline and figured it out.

Reply:
https://github.com/Microsoft/AppCenter-SDK-Android/pull/970#discussion_r261454407